CONTRIBUTING.rst

```diff
@@ -122,19 +152,24 @@ Dependencies for production Pontoon are in ``requirements.txt``. Development dep
 
 Note that we use a specific format for our dependencies, in order to make them more maintainable. When adding a new requirement, you should add it to the appropriate section and group it with its sub-dependencies if applicable.
 
-For example, to add ``foobar`` version 5::
+For example, to add ``foobar`` version 5:
+
+.. code-block:: shell
 
-  $ hashin -r requirements.txt foobar==5
+    $ hashin -r requirements.txt foobar==5
 
 Then open ``requirements.txt`` and move the added dependencies to:
+
 * the first section if it has no other requirements
 * the second section if it has sub-dependencies, and add all its dependencies there as well.
 
 That format is documented more extensively inside the ``requirements.txt`` file.
```

Comment:
Should we explain adding frontend requirements here, as tedious as they are ATM?

Reply:
Hah, well, here's my answer: https://github.com/mozilla/pontoon/pull/1212 :smile:

Reply:
Ah, great! :)